### PR TITLE
fix(LLD): Retry CLS installation after locking & unlocking device

### PR DESCRIPTION
**Pull request by fAnselmi-Ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

LLD will retry CLS installation on Stax and Flex after locking & unlocking device while CLS image is being uploaded

### ❓ Context

- **JIRA or GitHub link**: [LIVE-13238](https://ledgerhq.atlassian.net/browse/LIVE-13238)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-13238]: https://ledgerhq.atlassian.net/browse/LIVE-13238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by smartine-ledger — APPROVED** [review]
👍 . Is there no new UT because this code is untestable isn't it?

**Review by valpinkman — DISMISSED** [review]

File: apps/ledger-live-desktop/src/renderer/components/CustomImage/CustomLockScreenDeviceAction/index.tsx [hunk 1]
```diff
@@ -115,8 +117,9 @@ const CustomImageDeviceAction: React.FC<Props> = withRemountableWrapper(props =>
 
   const handleRetry = useCallback(() => {
     if (isRefusedOnStaxError) onTryAnotherImage();
+    if (status.onRetry) status.onRetry();
```
Comment (ofreyssinet-ledger) [inline apps/ledger-live-desktop/src/renderer/components/CustomImage/CustomLockScreenDeviceAction/index.tsx:120]: [SHOULD] use `else if` to be safe

File: apps/ledger-live-desktop/src/renderer/components/CustomImage/CustomLockScreenDeviceAction/index.tsx [hunk 2]
```diff
@@ -105,8 +105,10 @@ const CustomImageDeviceAction: React.FC<Props> = withRemountableWrapper(props =>
     if (error instanceof ImageCommitRefusedOnDevice) {
       dispatch(clearLastSeenCustomImage());
     }
-    onError && onError(error);
-  }, [dispatch, error, onError]);
+    if (!isRefusedOnStaxError) {
+      onError && onError(error);
+    }
+  }, [dispatch, error, onError, isRefusedOnStaxError]);
```
Comment (ofreyssinet-ledger) [inline apps/ledger-live-desktop/src/renderer/components/CustomImage/CustomLockScreenDeviceAction/index.tsx:111]: [ASK] Why this change ? What happens now when the user refuses the picture on the device ?

**Review by aussedatlo — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/components/CustomImage/CustomLockScreenDeviceAction/index.tsx (hunk 2 above)
Comment (fAnselmi-Ledger) [reply apps/ledger-live-desktop/src/renderer/components/CustomImage/CustomLockScreenDeviceAction/index.tsx]: @ofreyssinet-ledger I simplified the logic now as the prev. condition was redundant

**Review by aussedatlo — APPROVED** [review]

**Review by themooneer — APPROVED** [review]
